src/routes/paths.ts

```diff
@@ -143,12 +149,24 @@ export const getMainNavigation = ({
   return mainItems;
 };
 
+export const getAdminNavigation = (): INavItem[] => [
```

Comment:
I know you're probably just following the existing pattern, but these data structures are small & simple enough that they can just be embedded directly in the components that use them.

Reply:
@MarcogodoyV this^ is the last thing to clean up, & then it looks good to merge